web/src/components/dashboard/ProjectDetails.tsx

```diff
@@ -83,7 +83,7 @@ const ProjectDetails: React.FC<any> = (props) => {
 
   return (
     <>
-      <Box style={{ overflow: 'auto' }}>
+      <Box style={{}}>
```

Comment:
also do you think this is the best solution? do we leave the right 0 margins while the left side has some margin(the table is not at the middle of the page.)

Reply:
I don't think it's possible to show the entire table in the window if it's minimized. The image may not have been very clear, but having no margin on the right side indicates that there is more to the table that the viewer can scroll over to.

Reply:
I have pulled this branch to my local and tested it. and made comment based on what I saw.